File: redis/python/get.py
import redis

# Create Redis client
redisClient = redis.Redis(host='localhost', port=6379, username='default', password='', decode_responses=True)

# Set value for 'firstkey'
# Command: set firstkey "First Key value set by Python Script"
# Result: OK
redisClient.set('firstkey', 'First Key value set by Python Script')

# Get value of 'firstkey'
# Command: get firstkey
# Result: "First Key value set by Python Script"
firstKey = redisClient.get('firstkey')

print('Value of the "firstkey": ' + firstKey)

# Get value of 'wrongkey', this key does not exist
# should return an empty response (None in case of python)
# Command: get wrongkey
# Result: nil
wrongKey = redisClient.get('wrongkey')

print('Value of the "wrong": ' + str(wrongKey))

File: redis/ruby/get.rb
require 'redis'

redis = Redis.new(host: "localhost", port: 6379)

# Set a key/value
# Command: set firstkey "some value"
# Result: OK
commandResult = redis.set("firstkey", "some value")

print("Command: set firstkey \"some value\" | Result: ", commandResult, "\n")


# Check the value of key firstkey
# Command: get firstkey
# Result: "some value"
commandResult = redis.get("firstkey")

print("Command: get firstkey | Result: ", commandResult, "\n")


# Check the value of key wrongkey (which does not exist in database)
# Command: get wrongkey
# Result: nil
commandResult = redis.get("wrongkey")

print("Command: get wrongkey | Result: ", commandResult, "\n")
